vscode-extension/README.md

```diff
@@ -36,22 +38,35 @@ The following elements of a schema file are highlighted:
 
 - Model Blocks (`model`)
 - Config Blocks (`datasource` and `generator`)
-- Enum Blocks (`enum`)
+- Enum Blocks (`enum`) (not supported yet)
 - Double and triple comments (`//` and `///`)
 
-TODO appropriate screenshot that shows all functionality
+![Alt text](./images/syntaxHighlighting.png?raw=true "Syntax Highlighting example")
```

Comment:
Nitpick: The images should have an appropriate alternative text as well. (applies to all images)